client/game.cpp

```diff
@@ -106,13 +110,16 @@ namespace tec {
 	}
 
 	void Game::Update(double delta, double mouse_x, double mouse_y, int window_width, int window_height) {
+		// TODO: a better representation of commands so we can send them less often
+		const double COMMAND_RATE = 1.0 / 30.0;
```

Comment:
Should this be decoupled form the update rate?

Reply:
Ideally, there would be one command per frame, but instead of sending them that frequently, they get batched up with a timestamp or something and sent in bursts. This should be decoupled from the update rate, and set fairly high to maintain smooth control, but since we don't have batching, this is the compromise setting